robot/web/web.go

```diff
@@ -353,6 +361,9 @@ func (svc *webService) Close(ctx context.Context) error {
 	if svc.modServer != nil {
 		err = svc.modServer.Stop()
 	}
+	if svc.streamServer != nil {
+		utils.UncheckedError(svc.streamServer.Close())
```

Comment:
There is a leak related to not calling this. I'm assuming not closing the stream server was an accidental omission

Reply:
I would assume so, too; and, it sounds like you were the first person to add a test that actually exercised the path to leak this goroutine.

Reply:
The test existed, I was just the first person to create a code path that spins up the stream server part of the robot's web service in testing.